radicle/src/cob/patch.rs

```diff
@@ -479,6 +488,19 @@ pub struct Review {
     pub timestamp: Max<Timestamp>,
 }
 
+impl Serialize for Review {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::ser::Serializer,
+    {
+        let mut state = serializer.serialize_struct("Review", 3)?;
+        state.serialize_field("verdict", &self.verdict())?;
```

Comment:
notice that `inline` is missing. needed?

Reply:
Not for now.